.github/workflows/automerge.yml

```diff
@@ -10,6 +10,10 @@ on:
       - synchronize
       - unlabeled
       - unlocked
+    paths-ignore:
+      - README.md
+      - CONTRIBUTING.md
+      - docs/**
```

Comment:
Revert this please. If I'm not wrong, I might still be able to use automerge for doc updates.